scout/parse/variant/transcript.py

```diff
@@ -20,25 +20,16 @@ def parse_transcripts(raw_transcripts, allele=None):
         transcript = {}
         # There can be several functional annotations for one variant
         functional_annotations = entry.get("CONSEQUENCE", "").split("&")
+        # XXX: functional annotations is set again in end of function, is that correct?
         transcript["functional_annotations"] = functional_annotations
```

Comment:
Agreed, looks really useless? The data is stored in functional_annotations, and there should be no way this function terminates in an orderly fashion before the point where the transcript reference to it is overwritten! Let's remove this assignment.

Vulture also notes a few unused above - seems safe to fix.